db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Retirement/StateMachines/Methods.class/__methods__/microsoft_pre_retirement.rb

```diff
@@ -0,0 +1,10 @@
+#
```

Comment:
@gmcculloug @bzwei 
Since there is nothing vendor specific about the microsoft_pre_retirement method, maybe we should rename it pre_retirement.